### ISSUE TITLE
Add README to document conversion and technology mapping command line

### ISSUE THREAD

**Issue by cjmyers** [MyersResearchGroup/iBioSim#445]

**Comment by tramyn** [comment]
@cjmyers Looking at the title of this issue, you want me to create a README for the conversion package to run conversion and tech map, not for Analysis and Learn right?

**Comment by cjmyers** [comment]
Yes, we need READMEs to document the command line execution for all four, but Leandro can handle analysis and learn, you can take conversion and techmap.

**Comment by tramyn** [comment]
Update: A [README.md](https://github.com/MyersResearchGroup/iBioSim/tree/master/conversion) was created for converter package to build Converter.java and VPR model generation. Right now, the README.md has instruction to build Converter.java and not instructions to build the JARs for SBOL2SBML and SBML2SBOL. 

Still need to create a README.md for Tech. Map.

**Comment by cjmyers** [comment]
The readme’s should also include instructions for running the JAR at the command line.

**Comment by cjmyers** [comment]
Fix type on --rsbol

Add example to convert SBML to SBOL and another to convert SBOL to SBML.

Add example for SBML techmap and SBOL techmap

**Comment by tramyn** [comment]
README.md now has example to run both conversion

**Comment by cjmyers** [comment]
Examples can come later.
